subjects/hangman/hangman-classic/README.md

```diff
@@ -2,17 +2,16 @@
 
 This project is based on the famous HangMan Game
 
-You will need to create a public repository with the name `hangman`
+You will need to create a private repository with the name `hangman-classic`
 
 ### Notions
-- [Golang Documentation: net](https://golang.org/pkg/net/)
 - [Golang Documentation: ioutil](https://golang.org/pkg/ioutil/)
 - [Golang Documentation: rand](https://golang.org/pkg/math/rand/)
 
 ### Instructions
 
 Create a program `hangman` that will take a file as parameter. 
-This file contains a bunch of words with which the program will play. Each word is separated with a newline.
+Create a file `words.txt` witch contains a bunch of words with which the program will play. Each word is separated with a newline `\n`.
```

Comment:
"which", not "witch" sinon c'est la sorcière

Reply:
oh oui, c'est pas moi qui est écrit cette partie mdrr